src/App.js

```diff
@@ -1,16 +1,132 @@
 import React from "react";
 import logo from "./logo.png";
 import "./App.css";
+import axios from "axios";
 
 class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      cityInputValue: "",
+      city: "",
+      weather: {},
+      weatherIcon: "",
+      coordinates: { lat: "", lon: "" },
+      forecastWeather: [],
+    };
+
+    this.handleChange = this.handleChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  handleChange(event) {
+    this.setState({ cityInputValue: event.target.value });
+  }
+
+  handleSubmit = (event) => {
+    event.preventDefault();
+    axios
+      .get(
+        `https://api.openweathermap.org/geo/1.0/direct?q=${this.state.cityInputValue}&limit=1&appid=b19b32221939a66675b01f5452af913c`
+      )
+      // City geo data is in response.data[0]
+      // Arrow functions with no curly braces return value after arrow
+      .then((response) => {
+        this.setState({
+          coordinates: { lat: response.data[0].lat, lon: response.data[0].lon },
+        });
+        return response.data[0];
+      })
+      .then((cityGeoData) =>
+        axios.get(
+          `https://api.openweathermap.org/data/2.5/weather?lat=${cityGeoData.lat}&lon=${cityGeoData.lon}&appid=b19b32221939a66675b01f5452af913c&units=metric`
+        )
+      )
+      .then((response) => {
+        const { data: weatherData } = response;
+        console.log(this.state.coordinates);
+        this.setState({
+          city: this.state.cityInputValue,
+          weather: weatherData.main,
+          weatherIcon: weatherData.weather[0],
+          cityInputValue: "",
+        });
+      })
+      .then(() =>
```

Comment:
Since you are not returning anything in the previous .then block, I recommend moving the following code to its own promise, basically not using a .then, but ending with the previous one and then running another axios.get in the following lines of the submit handler. This gives a better overview and will not create nested promises, which are often the reason for bugs.